api/GPUQueue.json

```diff
@@ -344,6 +344,47 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "unique_command_buffers": {
+          "__compat": {
+            "description": "Validates that submitted command buffers are unique.",
+            "tags": [
+              "web-features:webgpu"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "126",
+                "notes": "Currently supported on ChromeOS, macOS, and Windows only."
```

Comment:
Nit: It might be nice to explicitly mention the platforms where the feature is not available, but given that there are already 405 notes like this across the GPU features, this is probably not the right place to start this.

Reply:
Yeah...